e2e/cypress/integration/integrations/incoming_webhook/description_field_for_incoming_and_outgoing_webhooks_can_hold_500_characters_spec.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+let testTeam;
+
+describe('Integrations', () => {
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+
+            // # Visit the Incoming Webhooks add page
+            cy.visit(`/${team.name}/integrations/incoming_webhooks/add`);
+        });
+    });
+
+    it('MM-T636 Description field for incoming webhooks can hold 500 characters', () => {
+    // * Check Incoming Webhooks - Enter 503 characters into the field then assert there are only 500 and not 503
+        cy.get('#description').clear().type('12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123').should('have.value', '12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890').and('not.have.value', '12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123');
```

Comment:
For better readability, can you instead assign the 500 char text into a variable and then use the same in the test steps. For example:

```
const description = '1234.......';
cy.get('#description').clear().type(`${description}123`).should('have.value', description);
```
Also just verifying `should('have.value')` is enough. It's not necessary to validate `should('not.have.value')` as we are validating the complete value in the first validation itself.